.github/workflows/github-workflows.yaml

```diff
@@ -0,0 +1,21 @@
+name: python-backend-workflows
+
+on:
+  pull_request:
+    types:
+      - opened
+      - closed
+      - synchronize
+
+jobs:
+  #   lint-test: to be determined
+
+  markdown-check:
+    uses: ai-cfia/github-workflows/.github/workflows/workflow-markdown-check.yml@main
+    with:
+      config-file-path: ".mlc_config.json"
+    secrets: inherit
+
+  repo-standard:
```

Comment:
Done. Because the packages are independent and in separate folders, the reusable lint-test workflow would not work. I created this temporary one that detects packages based on presence of `requirements.txt` then cd into each and performs the lint-tests. I'll leave it to devops to come up with a more robust et reusable solution.